backend/fms_core/services/dataset.py

```diff
@@ -176,7 +188,9 @@ def set_experiment_run_lane_validation_status(experiment_run_id: int, lane: int,
 
     if not errors:
         for dataset in Dataset.objects.filter(experiment_run_id=experiment_run_id, lane=lane): # May be more than one dataset due to projects
+            is_status_revocation = False
             for readset in Readset.objects.filter(dataset=dataset).all():
```

Comment:
Can you bring the `is_status_revocation` declaration closer to its actual usage?

Reply:
It might not even need to be declared.